docs/admin/observability/alerting.mdx

```diff
@@ -7,31 +7,31 @@ Alerts can be configured to notify site admins when there is something wrong or
 Alerts fall in one of two severity levels:
 
 - <span class="badge badge-critical">critical</span>: something is _definitively_ wrong with Sourcegraph, in a way that is very likely to be noticeable to users. We suggest using a high-visibility notification channel for these alerts.
-  - **Examples:** Database inaccessible, running out of disk space, running out of memory.
+  - **Examples:** Database inaccessible, running out of memory, gitserver volume 90% full.
   - **Suggested action:** Page a site administrator to investigate.
-- <span class="badge badge-warning">warning</span>: something _could_ be wrong with Sourcegraph. We suggest checking in on these periodically, or using a notification channel that will not bother anyone if it is spammed. Over time, as warning alerts become stable and reliable across many Sourcegraph deployments, they will also be promoted to critical alerts in an update by Sourcegraph.
-  - **Examples:** High latency, high search timeouts.
+- <span class="badge badge-warning">warning</span>: something _could_ be wrong with Sourcegraph, or may go wrong soon. We suggest checking in on these periodically, or using a notification channel that will not bother anyone if it is spammed. Over time, as warning alerts become stable and reliable across many Sourcegraph deployments, they will also be promoted to critical alerts in an update by Sourcegraph.
+  - **Examples:** High latency, high search timeouts, gitserver volume 80% full.
   - **Suggested action:** Email a site administrator to investigate and monitor when convenient, and please let us know so that we can improve them.
 
-Refer to the [alert solutions reference](https://docs.sourcegraph.com/admin/observability/alerts) for a complete list of Sourcegraph alerts, as well as possible solutions when these alerts are firing.
+Refer to the [alert solutions reference](https://docs.sourcegraph.com/admin/observability/alerts) for a complete list of Sourcegraph alerts, as well as possible solutions when these alerts are firing$$.
```

Comment:
@marcleblanc2 we should probably update this to point to the new docsite @ https://sourcegraph.com/docs

Reply:
I think this should point to the old one for now actually, because this new one doesn't sync changes from generated docs like the linked page, so linking to the https://sourcegraph.com/docs equivalent here will point to outdated contents

Reply:
I agree with both of you again, and I anticipate that cleaning up all the old links will happen as one big push when we shut down the old doc site.

Reply:
good to know! also https://sourcegraph.com/docs/admin/observability/alerts doesn't even exist...

Reply:
good to know! also https://sourcegraph.com/docs/admin/observability/alerts doesn't even exist...